### ISSUE TITLE
Purchase Receipt shows To Bill even if Purchase Invoice exists

### ISSUE THREAD

**Issue by bcornwellmott** [frappe/erpnext#9911]
The purchase receipt isn't grabbing the correct data for status if the Purchase Invoice was made directly from the Purchase Order (instead of through the Purchase Receipt).

**Comment by bcornwellmott** [comment]
It looks like the update_billing_status() for the purchase receipt doesn't get called when the Purchase Invoice gets created / submitted. I'm not sure how to fix this? Should the Purchase Order have a piece of code that calls update_billing_status() for all linked purchase receipts with docstatus = 1??

**Comment by rohitwaghchaure** [comment]
Hi @bcornwellmott 

I have checked on local account and it's working properly.
I have created Purchase order first then created purchase receipt against order and then purchase invoice against the purchase order. Even the flow PO > PI and then PR from PO also working properly.

**Comment by bcornwellmott** [comment]
You're right! I checked on my development server and it seems to work. That doesn't change that my production system is showing purchase receipts with "To Bill" even though the purchase order is showing Completed.

It looks like the issue is to do with orders that are split up (several invoices or several purchase receipts). It could be that part of the order was invoiced or received from the PR/PI (e.g. PR > Make > PI), and the other part received or invoiced from the purchase order (e.g. PO > Make > PI). I realized I can close the purchase receipts so it isn't a big deal. I can't seem to replicate it on my test server.
